apps/enterprise-knowledge-retrieval/README.md

```diff
@@ -0,0 +1,36 @@
+# Enterprise Knowledge Retrieval
+
+This repo is a deep dive on Enterprise Knowledge Retrieval, which aims to take some unstructured text documents and create a usable knowledge base application with it.
```

Comment:
is repo the right term? I think of the Cookbook as a the repo and this as an example app